helm/vault-exporter/values.yaml

```diff
@@ -7,37 +7,7 @@ pod:
     id: 1000
   group:
     id: 1000
-project:
-  name: "vault-exporter"
-# Resource names are truncated to 47 characters. Kubernetes allows 63 characters
-# limit for resource names. When pods for deployments are created they have
-# additional 16 characters suffix, e.g. "-957c9d6ff-pkzgw" and we want to have
-# room for those suffixes.
-#
-# NOTE: All values under resource key need to be used with `tpl` to render them
-# correctly in the templates. This is because helm doesn't template values.yaml
-# file and it has to be a valid json. Example usage:
-#
-#     {{ tpl .Values.resource.default.name . }}.
-#
 resource:
-  default:
-    name: "vault-exporter"
-    namespace: "vault-exporter"
-  np:
-    name: "vault-exporter-network-policy"
-  port:
+  service:
     port: "9410"
     protocol: "TCP"
-  psp:
-    name: "vault-exporter-psp"
-  pullSecret:
-    name: "vault-exporter-pull-secret"
-    namespace: "vault-exporter"
-
-vault:
-  address: https://example.com
```

Comment:
This moved to ci/default-values.yaml. I prefer to fail with non existing value than having it running with BS value.